javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,55 @@
+//The Game
+let body = document.querySelector("body");
+let gameTimerInput = document.querySelector(".game-timer");
+let gameStartBtn = document.querySelector(".game-starter");
+let sSide = document.querySelector(".s-part");
+let lSide = document.querySelector(".l-part");
+let resultNotification = document.querySelector(".result-notification");
+let restartBtn = document.querySelector(".restart");
+let sCounter = document.querySelector(".s-counter");
+let lCounter = document.querySelector(".l-counter");
+
+gameStartBtn.addEventListener("click", function startingFunction() {
+  let l = 0;
+  let s = 0;
+
+  if (gameTimerInput.value == 0 || gameTimerInput.value == " ") {
+    resultNotification.innerHTML = "Enter How many seconds you want to play";
+  } else {
+    body.addEventListener("keydown", function gaming(press) {
```

Comment:
Great!  Maybe in here you would probably want to use "keyup" event, as "keydown" allows players to cheat if they keep pressing the key. Keydown fires the event while the key is down it keeps running the event again and again, while keyup fires the event on key release.